### PR TITLE
LPS-171009 Add objectDefinitionERC and Create Endpoints GET and POST for ObjectValidationRules by ObjectDefinitionERC

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-171009

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
Hey @p-albuquerque , if the CI errors are not related, please send this PR directly to Brian. Thanks!

File: modules/apps/object/object-admin-rest-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -872,6 +874,71 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectRelationship"
             tags: ["ObjectRelationship"]
+    "/object-definitions/by-external-reference-code/{objectDefinitionExternalReferenceCode}/object-validation-rules":
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-admin-rest-impl/rest-openapi.yaml:877]: As we just discussed, please change the path parameter name from `objectDefinitionExternalReferenceCode` to `externalReferenceCode`

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Review by guilhermedcamacho — APPROVED** [review]
LGTM!

**Comment by guilhermedcamacho** [comment]
Hey @p-albuquerque , the code looks good. Please solve the conflicts and run CI again. Thanks!

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
Resent in: https://github.com/brianchandotcom/liferay-portal/pull/128055

**Pull request closed without merging**